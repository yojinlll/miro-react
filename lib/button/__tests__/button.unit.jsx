import * as renderer from 'react-test-renderer'
import React from 'react'
import Button from '../button'

describe('button', ()=>{
  it('存在', ()=>{
    const json = renderer.create(<Button>button</Button>).toJSON()
    expect(json).toMatchSnapshot()
  })
})