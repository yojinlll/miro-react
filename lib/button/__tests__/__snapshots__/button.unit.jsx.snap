// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 存在 1`] = `
<button
  className="miro-button"
>
  button
</button>
`;
